### ISSUE TITLE
[F.Req] Please add the desktop app to the AUR.

### ISSUE THREAD

**Issue by yum13241** [PenguinMod/penguinmod.github.io#25]
The **AUR** is the **A**rch **U**ser **R**epository. Basically it lets users of Arch Linux and derivatives download and install this with **AUR** helpers like `yay`, or the boring manual way.

[Here's an wiki page that'll tell you how to do it.](https://wiki.archlinux.org/title/Creating_packages) You can also just base your PKGBUILD off of TurboWarp's.

TL;DR: Easy updates for Arch users, please.

**Comment by JeremyGamer13** [comment]
i could try adding this for launcher once its done but uhh, the desktop app doesnt exist yet?
PenguinMod-Desktop is just an unusable repo iirc (not sure why we havent archived or deleted it) and PenguinMod-Launcher is in development

**Comment by yum13241** [comment]
Oh. Thanks anyway. I guess I'll have to deal with browser version for now, lol (it's slow)

**Comment by sometgirldotonline** [comment]
we need a desktop app, is it possible to run penguin mods scratch-gui, ive been trying and it wont run. do i need other repos or something?

**Comment by sometgirldotonline** [comment]
or atleast introduce a SW so it can run offline as a PWA

**Comment by RedMan13** [comment]
> Oh. Thanks anyway. I guess I'll have to deal with browser version for now, lol (it's slow)

a desktop app wont fix that...

**Comment by sometgirldotonline** [comment]
> > Oh. Thanks anyway. I guess I'll have to deal with browser version for now, lol (it's slow)
> 
> a desktop app wont fix that...

yes it can, it'll fix a slow internet connection, which is a burden for myself in Australia(we have terrible internet, all of australia)

**Comment by RedMan13** [comment]
isnt there argument that the app runs slowly though...

**Comment by sometgirldotonline** [comment]
> isnt there argument that the app runs slowly though...

?

**Comment by sometgirldotonline** [comment]
> The **AUR** is the **A**rch **U**ser **R**epository. Basically it lets users of Arch Linux and derivatives download and install this with **AUR** helpers like `yay`, or the boring manual way.
> 
> [Here's an wiki page that'll tell you how to do it.](https://wiki.archlinux.org/title/Creating_packages) You can also just base your PKGBUILD off of TurboWarp's.
> 
> TL;DR: Easy updates for Arch users, please.
its a nodejs app, can just be run from source and updated by pulling latest with git

**Comment by RedMan13** [comment]
:skull:

**Comment by LibraryOfStuff** [comment]
If you mean arch like I think you do... 🐧💀

**Comment by LibraryOfStuff** [comment]
> i could try adding this for launcher once its done but uhh, the desktop app doesnt exist yet? PenguinMod-Desktop is just an unusable repo iirc (not sure why we havent archived or deleted it) and PenguinMod-Launcher is in development

So basically, is the PenguinMod-Launcher just PenguinMod desktop app but more modern and less old and clanky?

**Comment by Gen1xLol** [comment]
why is this issue from 2023 still open

**Comment by Steve0Greatness** [comment]
> If you mean arch like I think you do... 🐧💀

They say in the post that they mean the Arch User Repository.


> So basically, is the PenguinMod-Launcher just PenguinMod desktop app but more modern and less old and clanky?

Sorry what? What do you even mean by that?

**Comment by Steve0Greatness** [comment]
Anyway, this should probably be closed because PM doesn't currently have an official desktop app, and didn't even at the time of this being posted so this has essentially always been a dud.
